src/index.mako.html

```diff
@@ -230,6 +230,8 @@
     </div>
 % endif
 
+    <div ga-controls3d ga-controls3d-ol3d="ol3d" ng-if="ol3d" ng-show="globals.is3dActive"></div>
```

Comment:
We can use `ng-if="globals.is3dActive && ol3d"` and no `nf-if` but the components will be re-created every time the 3d mode is activated.
I've updated the code.